packages/angular/golden/clr-angular.d.ts

```diff
@@ -889,7 +889,7 @@ export declare class ClrDateInput extends WrappedFormControl<ClrDateContainer> i
     onValueChange(target: HTMLInputElement): void;
     setFocusStates(): void;
     triggerValidation(): void;
-    static ngAcceptInputType_date: Date | null;
+    static ngAcceptInputType_date: Date | null | string;
```

Comment:
That's not the most supportable feature I've seen :( 
Still don't have a clue why we would need it.